### PR TITLE
Add History Screen for Viewing Messaging, Voice Call, and Video Call History

### LINKED ISSUE #81: Feature: Online Consultation History 08(78,81,85)

### PR DESCRIPTION
**Pull request by KeynesBizimana**
### What does this PR do?

This PR adds a History screen that allows users to view their messaging history, voice call history, and video call history.

### Description:

Added a new History screen to display the messaging, voice call, and video call history of users.

### Changes Made:

Developed the History screen UI to categorize and display different types of communication history.
Implemented functionality to fetch and display messaging history.
Implemented functionality to fetch and display voice call history.
Implemented functionality to fetch and display video call history.
Updated navigation to include the new History screen.
### Screenshots:

Screenshot_20240522_150651_Expo Go
Screenshot_20240522_150639_Expo Go
Screenshot_20240522_150712_Expo Go

### CODE REVIEW

**Review by mr3nz1 — APPROVED** [review]
Looks good

**Review by Irirwanirira — APPROVED** [review]
Good job

File: app/(app)/History/MessageHistory/index.tsx [hunk 1]
```diff
@@ -1,10 +1,199 @@
...
+        </View>
+        <View
+          style={{
+            flexDirection: "row",
+            justifyContent: "space-around",
+            alignItems: "center",
+            width: "100%",
+            //  borderBottomWidth: 1,
+            // borderBottomColor: "#D3D3D3",
+            // paddingBottom: 10,
```
Comment (Nkbtemmy) [inline app/(app)/History/MessageHistory/index.tsx]: Remove commented codes

**Review by Nkbtemmy — CHANGES_REQUESTED** [review]

File: app/(app)/History/MessageHistory/index.tsx (hunk 1 above)
Comment (KeynesBizimana) [reply app/(app)/History/MessageHistory/index.tsx]: Done !!!

**Review by Nkbtemmy — APPROVED** [review]

**Pull request merged**